app/components/shared/Icon/RevealableDownChevron.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import Icon from '../Icon';
+
+const RotatableIcon = styled(Icon)`
+  transform: rotate(${(props) => props.revealed ? -90 : 90}deg);
+  trasform-origin: center 0;
+  transition: transform 300ms;
+  will-change: transform;
+  width: ${(props) => props.size}px;
+  height: ${(props) => props.size}px;
+
+  polyline {
+    stroke-width: ${(props) => props.strokeWidth}px;
+  }
+`;
```

Comment:
I feel like you could just export this `RotatableIcon` itself and have the same functionality, with `defaultProps` specified. Bonus points for attaching `propTypes`!